cmake/README.md

```diff
@@ -0,0 +1,52 @@
+Building with CMake
+-------------------
+
+The quickest way to build and install panda with CMake is to run:
+```sh
+mkdir build && cd build
+cmake ..
+make
+[sudo] make install
+```
+
+To configure CMake, it is recommended to use cmake-gui (`cmake-gui .`),
+however it is also possible to configure it entirely through CMake's
+command-line interface; see `man cmake` for more details.
```

Comment:
I'll probably postpone this particular concern. Some kind of instruction on how to use the new CMake buildsystem is extremely important; however it might belong in the root `README.md` instead, and we should hold off until we ourselves know what the best practices are before we tell people what new habits to form.

To that end I'm considering just deleting this file. Thoughts?

Reply:
That's fine.  I do think we need to acknowledge the CMake stuff in some file somewhere, maybe just put a note in README.md that it's possible to experimentally build with CMake but that documentation is forthcoming.